packages/harpguru-core/src/hooks/use-sizes/use-sizes.ts

```diff
@@ -78,20 +77,35 @@ export const useSizes = (): SizeSchemes => {
   const legendWidth = columnWidth
   const zoomSlideWidth = useIsZoomedColumnBounds() === false ? 0 : columnWidth
 
-  const { length: groupCount } = useOctaveColumnGroups('harpface1', true)
+  // We need the sizing scheme to be as independant from the updates of the
+  // updates of the global properties as possible. If it isn't then we will
+  // see size updates after insignificant interactions. Getting an accurate
+  // measure of the number of octave groups is a waste of effort because it
+  // requires a lot of context and only very slightly improves the accuracy
+  // of this sizing seed. There are not many harps having fewer than 3 hole
+  // octave spreads. Most have 4, but using 3 doesnt disrupt the layout too
+  // much.
+  const pessimisticGroupColumnnEstimate = 3
+  const roughFullGroupCount = Math.ceil(
+    harpfaceColumnCount / pessimisticGroupColumnnEstimate
+  )
   const exteriorGutterCount = 2
-  const includingHarpFaceEdgesGutterCount = groupCount + exteriorGutterCount
+  const includingHarpFaceEdgesGutterCount =
+    roughFullGroupCount + exteriorGutterCount
   const dynamicWidthRequirements =
     longEdge /
     (columnWidth * harpfaceColumnCount +
       fragmentGutter * includingHarpFaceEdgesGutterCount +
       labelProtrusion +
       legendWidth +
       zoomSlideWidth)
+  const [fullInteractionMatrices] = useGlobal('activeInteractionMatrix')
```

Comment:
Moving the `fullInteractionMatrices` `useGlobal` call outside of the IIFE was unnecessary. The call itself should be unnecessary though. I think the TODO below doesn't go far enough. We should make the sizing scheme be a derived global property which is *only* dependent on the `viewableInteractionMatrices`.

Doing that would ensure that we can work towards the absolute minimum number of resize updates because we get to do the bailout return from the reducer and return the same object.

Reply:
Issue raised: https://github.com/js-jslog/harpguru/issues/134

Reply:
This will be dealt with later. The important thing for this particular pull request is that I haven't made the performance significantly worse than master. See this discussion for details and checks: https://github.com/js-jslog/harpguru/pull/133#discussion_r761758012